apps/install-app.md

```diff
@@ -72,8 +72,30 @@ Substitutions would be made as given in this example table:
 | {`api_url`}      | The base API URL, that uniquely identifies the merchant. The `api_url` differs for every merchant and has to be stored in the app.  | https://creamyiceshop.com/rs/shops/CreamyIceShop |
 | {`return_url`}    | The URL which the merchant should be redirected to after the app installation. | https://creamyiceshop.com/epages/CreamyIceShop.admin/?ObjectID=17811&ViewAction=MBO-ViewAppDetails&appID=54f46f318732110bd85f41c7 |
 | {`access_token_url`} | The URL to obtain the `access_token`. | https://creamyiceshop.com/rs/shops/CreamyIceShop/token. |
+| {`signature`}      | The signature required to validate the `access_token_url`. The signature is calculated with the `code`, `access_token_url` and `client_secret`.    | jEPRUggebJDBsEnl1%2FpHlMUBxPbsELQihEVzbx2pFlM%3D |
```

Comment:
better change to "The signature is an message authentification code. It is calculated with the `code`, `access_token_url` and `client_secret`."